cart/cart.go

```diff
@@ -0,0 +1,81 @@
+package cart
+
+import (
+	"checkout_system/models"
+	"checkout_system/upload"
+	"github.com/sirupsen/logrus"
+)
+
+type ProductCart struct {
+	Products []models.Product `json:"products"`
```

Comment:
I think it might be better to use a map here 
```
Products [string]models.Product
```
where string is SKU. This should allow us to speed up products search process